src/main/editDelete/liveProcess.ts

```diff
@@ -0,0 +1,42 @@
+import { Transcription, Word } from 'sharedTypes';
+
+/**
+ * calculateTime calculates the outputStartTimes of a word based on the
+ * current state of all words in a Transcript. It takes into account edits/deletes
+ * @param word word in Transcript
+ * @param i the current index position in the words array
+ * @param words an array of words from Transcript
+ * @returns the input word with an updated outputStartTime
+ *
+ */
+
+const calculateTime = (word: Word, i: number, words: Word[]): Word => {
+  if (!word.deleted) {
+    if (i === 0) {
+      word.outputStartTime = 0;
+    } else {
+      let j = i - 1;
```

Comment:
Like Liam said, a comment would be great here. Might be good to give j a descriptive name as well If you can.

Reply:
j has been renamed to nextNotDeleted, comments have been provided explaining the algorithm.